### DISCUSSION TITLE
music/projects/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/music#12] [General]
# music/projects/

http://shanenull.com/music/projects/

**Comment by shane0** [comment]
https://youtu.be/d4sw9tsxXiM?si=cQ3MN05qI2EE2EdE

**Comment by shane0** [comment]
# theory page

| Genre | Cinematic Music Scale Types |
|---|---|
| Action | Major, Minor, Pentatonic |
| Adventure | Major, Phrygian, Lydian |
| Comedy | Major, Dorian, Ionian |
| Drama | Minor, Phrygian, Aeolian |
| Fantasy | Major, Harmonic minor, Melodic minor |
| Horror | Minor, Phrygian, Locrian |
| Sci-Fi | Major, Harmonic minor, Mixolydian |
| Thriller | Minor, Phrygian, Aeolian |

**Reply by shane0** [reply]
```markdown
| Genre | Cinematic Music Scale Types | 7th Interval Chords |
|---|---|---|
| Action | Major, Minor, Pentatonic | Major 7, Minor 7, Dominant 7 |
| Adventure | Major, Phrygian, Lydian | Major 7, Minor 7, Dominant 7 |
| Comedy | Major, Dorian, Ionian | Major 7, Minor 7, Dominant 7 |
| Drama | Minor, Phrygian, Aeolian | Minor 7, Half-diminished 7, Dominant 7 |
| Fantasy | Major, Harmonic minor, Melodic minor | Major 7, Minor 7, Dominant 7 |
| Horror | Minor, Phrygian, Locrian | Minor 7, Half-diminished 7, Dominant 7 |
| Sci-Fi | Major, Harmonic minor, Mixolydian | Major 7, Minor 7, Dominant 7 |
| Thriller | Minor, Phrygian, Aeolian | Minor 7, Half-diminished 7, Dominant 7 |
```
